src/v/redpanda/admin_server.cc

```diff
@@ -3442,9 +3455,20 @@ void admin_server::register_usage_routes() {
           return _usage_manager
             .invoke_on(
               kafka::usage_manager::usage_manager_main_shard,
-              [](kafka::usage_manager& um) { return um.get_usage_stats(); })
-            .then([include_open](auto total_usage) {
-                return raw_data_to_usage_response(total_usage, include_open);
+              [this](kafka::usage_manager& um) {
+                  const bool is_leader = _controller->is_raft0_leader();
```

Comment:
Let's leave raft0 leadership out of this; let the API client hit a different path for  cluster-wide data, and they can decide which node they'd like to query it on.

Reply:
I agree with the comments on the JSON schema above but, it must be the case that only one node calls the cloud_reduction method every window close interval. 

We could keep things as they are (only performing the cloud stats logic on the controller leader) then have a new API to hit this which redirects to leader if called via a peer. Having a distinctly new endpoint with a new response type would also solve the issues mentioned in discussions above.